examples/misc/raptor.cfg

```diff
@@ -1,48 +1,49 @@
 
 {
-    "workload"       : {
-        # workload produce time in sec
-        "runtime"    : 60,
-        "timeout"    : null
-    },
-
-    "cpn"            :    4,
-    "gpn"            :    0,
-    "n_masters"      :    1,
-    "n_workers"      :    1,
-    "masters_pn"     :    1,
-    "nodes_pw"       :    1,
+    # resource configuration
+    "cores_per_node"  :   4,
+    "gpus_per_node"   :   0,
 
-    "nodes_rp"       :    0,
-    "tasks_rp"       :    1,
+    # raptor configuration
+    "n_masters"       :   1,
+    "n_workers"       :   1,
+    "masters_per_node":   1,
+    "nodes_per_worker":   1,
 
-    "nodes_agent"    :    0,
+    # extra nodes for rp tasks and for the rp agent
+    "nodes_rp"        :   0,
```

Comment:
nodes reserved for non-raptor tasks. should be made consistent with whether we are including non-raptor tasks

Reply:
```suggestion
    "nodes_rp"        :   1,
```